yoti-sdk-spring-boot-example/src/main/java/com/yoti/api/examples/springboot/YotiLoginController.java

```diff
@@ -212,9 +214,13 @@ public String identitySession(Model model) throws URISyntaxException {
                 .withRedirectUri(new URI("https://host/redirect/"))
                 .build();
 
-        ShareSession result = null;
+        ShareSessionQrRequest shareSessionQrRequest = ShareSessionQrRequest.builder().build();
+
+        ShareSession session = null;
+        ShareSessionQrCode sessionQrCode = null;
         try {
-            result = client.createShareSession(shareSessionRequest);
+            session = client.createShareSession(shareSessionRequest);
+            sessionQrCode = client.createShareSessionQrCode(session.getId(), shareSessionQrRequest);
         } catch (Exception ex) {
             LOG.error(ex.getMessage());
         }
```

Comment:
When doing assignments that need to be wrapped with a try/catch it's cleaner (easier to read and potentially to re-use) if the try/catch is extracted to a private method so that the try/catch is split from the assignment:
```java
ShareSession session = createShareSession(shareSessionRequest);
ShareSessionQrCode sessionQrCode = createShareSessionQrCode(session, shareSessionQrRequest);
```
and then do:
```java
private ShareSession createShareSession(...) {
    try {
        return ...
    } catch (...) {
        // handle exception
    }
}
```

That way the "technicalities" get pushed to a private method and the reader can "focus" on the business/main logic.